src/login/Login.js

```diff
@@ -16,8 +16,16 @@ export default class Login extends React.Component {
     super();
 
     let errorDiv = document.getElementById('login_errors');
+    let otpLength = 6;
     if (errorDiv) {
       this.errors = errorDiv.innerHTML;
+      try {
+        const json = JSON.parse(this.errors)
+        this.errors = json.text
+        otpLength = json.otpLength;
+      } catch (e) {
+        /* */
```

Comment:
Empty catch block looks strange, we should actually add a lint rule to prevent that.